### PR TITLE
Fix: visual bugs & issues with TimeSeriesChart

### PR DESCRIPTION
**Pull request by 3nvi**
## Background

This PR corrects a few edge cases with the rendering of a `TimeSeriesChart`, as well as its Tooltip

## Changes

- Make `units` optional within tooltip
- Merge typings into something exportable
- Fix edge cases with absolute positioning

## Testing

- `npm run test`

### CODE REVIEW

File: web/src/components/charts/TimeSeriesChart/TimeSeriesChart.tsx [hunk 1]
```diff
@@ -159,9 +160,16 @@ const TimeSeriesChart: React.FC<TimeSeriesChartProps> = ({
...
+          show: false,
+          formatter: ({ value }) => value[1].toLocaleString(),
           position: 'top',
+          fontSize: 11,
+          fontWeight: 'bold',
+          fontFamily: theme.fonts.primary,
           color: '#fff',
+          emphasis: {
+            show: !hideSeriesLabels,
+          },
```
Comment (3nvi) [inline web/src/components/charts/TimeSeriesChart/TimeSeriesChart.tsx:172]: Show labels only during hover. This was not in our original requirements, but it became mandatory after seeing this

Screen Shot 2020-11-23 at 2 26 05 PM

File: web/src/components/charts/TimeSeriesChart/TimeSeriesChart.tsx [hunk 2]
```diff
@@ -361,10 +375,10 @@ const TimeSeriesChart: React.FC<TimeSeriesChartProps> = ({
       <Box position="absolute" width="200px" ml={1} fontWeight="bold">
         {title}
       </Box>
-      <Box position="absolute" pl={hideLegend ? '50px' : '210px'} pr="50px" width={1}>
+      <Box position="absolute" left={0} pl={hideLegend ? '50px' : '210px'} pr="50px" width={1}>
         <Flex align="center" justify="space-between">
           {scaleControls && <ScaleControls scaleType={scaleType} onSelect={setScaleType} />}
-          <Box zIndex={5}>
+          <Box zIndex={5} ml="auto">
```
Comment (3nvi) [inline web/src/components/charts/TimeSeriesChart/TimeSeriesChart.tsx:381]: minor fixes for positioning the "RESET" button when the legend is missing

File: web/src/components/charts/TimeSeriesChart/TimeSeriesChart.tsx [hunk 3]
```diff
@@ -30,17 +30,17 @@ import useChartOptions from './useChartOptions';
 import ResetButton from '../ResetButton';
 import ScaleControls from '../ScaleControls';
 
-type SeriesMetadata = {
-  color?: keyof typeof Theme['colors'];
-};
+export type TimeSeries = (LongSeries | FloatSeries) & { color?: keyof typeof Theme['colors'] };
 
-export type SeriesDataMetadata = {
+export type TimeSeriesData = {
```
[the remark this answers is not in the document]
Comment (alexmylonas) [reply web/src/components/charts/TimeSeriesChart/TimeSeriesChart.tsx:35]: Nice call!

**Review by alexmylonas — APPROVED** [review]
LGTM!
